src/clr-angular/popover/tooltip/tooltip-trigger.ts

```diff
@@ -1,14 +1,32 @@
 /*
- * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import { Directive, HostListener } from '@angular/core';
+import { Directive, HostListener, Inject, Input } from '@angular/core';
 import { IfOpenService } from '../../utils/conditional/if-open.service';
+import { Subscription } from 'rxjs';
+import { UNIQUE_ID } from '../../utils/id-generator/id-generator.service';
 
-@Directive({ selector: '[clrTooltipTrigger]', host: { '[attr.tabindex]': '0', '[class.tooltip-trigger]': 'true' } })
+@Directive({
+  selector: '[clrTooltipTrigger]',
+  host: {
+    tabindex: '0',
+    '[class.tooltip-trigger]': 'true',
+    '[attr.aria-describedby]': 'tooltipId',
+    '[attr.aria-label]': 'tooltipLabel',
+    role: 'button',
+  },
+})
 export class ClrTooltipTrigger {
-  constructor(private ifOpenService: IfOpenService) {}
+  public ariaDescribedBy;
+  private subs: Subscription[] = [];
```

Comment:
I see no subscriptions here? Remove this and the onDestroy

Reply:
I had to add the subscription back into the trigger. The component will keep this.